modin/core/io/text/text_file_dispatcher.py

```diff
@@ -1020,8 +1022,10 @@ def _read(cls, filepath_or_buffer, **kwargs):
             nrows=None,
             compression=compression_infered,
         )
-
-        with OpenFile(filepath_or_buffer_md, "rb", compression_infered) as f:
+        storage_options = kwargs.get("storage_options") or {}
+        with OpenFile(
+            filepath_or_buffer_md, "rb", compression_infered, **storage_options
+        ) as f:
```

Comment:
```suggestion
        with OpenFile(
            filepath_or_buffer_md, "rb", compression_infered, **(kwargs.get("storage_options", None) or {}),
        ) as f:
```
To be consistent with other places of code and avoiding of using an extra variable